.github/workflows/recipe-catalog-change-trigger.yaml

```diff
@@ -1,17 +1,68 @@
 name: recipe-catalog-change-windows-trigger
 
 on:
-  pull_request:
-    paths:
-      - 'packages/backend/src/assets/ai.json'
+  workflow_run:
+    workflows: ["pr-check"]
+    types:
+      - completed
 
 jobs:
-  windows-e2e-recipe-change:
+  extract-context:
+    runs-on: ubuntu-latest
```

Comment:
I would suggest to set a fixed version [ubuntu-24.04](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md)

Reply:
yes, I missed that

Reply:
done